{{cookiecutter.project_slug}}/Makefile

```diff
@@ -107,16 +122,17 @@ training:  ### Run training job
 .PHONY: kill-training
 kill-training:  ### Stop training job
 	neuro kill $(TRAINING_NAME)
+	@echo kill-training: completed
 
 .PHONY: connect-training
 connect-training:  ### Execute shell to the training job
-	neuro exec $(TRAINING_NAME) bash
+	neuro exec --no-key-check $(TRAINING_NAME) bash
 
 .PHONY: jupyter
 jupyter:  ### Run jupyter job
 	neuro run \
 		--name $(JUPYTER_NAME) \
-		--preset gpu-small \
+		--preset cpu-small \
```

Comment:
Temporary change: currently, a `gpu-small` job cannot start within 10 minutes on `staging`. 
cc @mariyadavydova

Reply:
We can't merge this branch into master then.
I would better add `jupyter-cpu` (see #37) and test that command.

Reply:
please discuss with the team how we can solve the problem that gpu jobs are not always available. If it's unsolvable, @anaiden knows the hack how to allow circle ci to run jobs that don't print anything for more that 10 min